source/fresh-installation.rst

```diff
@@ -30,19 +30,26 @@ The configuration in ``/opt/conda/.condarc`` designates this location as the sec
 place conda should look for environments, after ``~/conda_envs``.
 
 Environments for data collection and data anlysis should be installed into
-``/opt/conda_envs``. In the second operating cycle of 2017, the commands to do
+``/opt/conda_envs``. In the second operating cycle of 2018, the commands to do
 this were:
 
 .. code-block:: bash
 
-    sudo conda create -p /opt/conda_envs/collection-17Q2.0 -c nsls2-tag -y collection
-    sudo conda create -p /opt/conda_envs/analysis-17Q2.0 -c nsls2-tag -y analysis
+    sudo conda create -n collection-2018-2.1 -c nsls2-tag -y collection
```

Comment:
The `-p` -> `-n` change should been applied to the section "Installing on a Personal Computer", not here. It's possible I left my comments in the wrong section.